### ISSUE TITLE
after internet loss -> low hash rate

### ISSUE THREAD

**Issue by xrtw** [Lolliedieb/lolMiner-releases#1481]
is it possible to make that if internet connection was rejected for a little, LHR stop unlocking card?

**Comment by xrtw** [comment]
https://i.gyazo.com/3c784cf3f701451c1fee318266edc845.png

**Comment by xiangcailai** [comment]
我也出现相同问题
![1649343988(1)](https://user-images.githubusercontent.com/103317183/162573960-86032a32-31c5-4abd-818c-9b17630d74ab.jpg)
看门狗重启矿机后才会正常

**Comment by xrtw** [comment]
> 

yes, so i just turned watchdog and it detects if after internet loss i'm having low hash rate on lolminer, it restarts

**Comment by xiangcailai** [comment]
>

看他下个版本会不会修复吧 老重启也不行 其他miner就不会这样子
